src/xenia/ui/imgui_drawer.h

```diff
@@ -14,6 +14,7 @@
 #include <cstdint>
 #include <memory>
 #include <vector>
+#include <optional>
```

Comment:
Hi, thank you for your fix — and welcome to Xenia!

`optional` should be between `memory` and `vector` alphabetically — overall, please apply `clang-format` (via `xb format` in our build script, after `git add`, and before another `git add` and `git commit`), which sorts includes among many other things, to all the changes submitted to Xenia.

Also, please add `[UI] ` to the beginning of the commit message, and prefer the present tense (what the commit itself does — "Add") as opposed to the past ("Added").

Reply:
Thanks for your feedback and sorry about those issues. I had totally missed them. I committed the file again after I ran `xb format` and I fixed the commit message.